[AspireE2E][GB18030] An error occurred when creating an Aspire project using Chinese characters as the project name.

**INSTALL STEPS**

1. Clean machine: Win11 x64 24h2 ENU
2. Install SDK 9.0.302
3. Install Aspire 9.3.1

**REPRO STEPS**

1. Open CMD, create an Aspire Starter App using following commands:
`dotnet new aspire-starter -o Aspire项目1`

**Note**: This issue also repro on Aspire 9.4.0-cobra.25367.16

**ACTUAL**
An error occurred when creating a project using Chinese characters as the project name
<img width="1095" height="503" alt="Image" src="https://github.com/user-attachments/assets/662f2129-9bf4-4dbf-9fac-9982cc6e414f" />

Aspire项目1 folder:
<img width="647" height="282" alt="Image" src="https://github.com/user-attachments/assets/a3677078-3957-4ef0-b97d-0e343b2eade8" />

Error Log:
Restoring C:\Users\v-reinawang\Aspire项目1\Aspire项目1.sln:
C:\Program Files\dotnet\sdk\9.0.302\NuGet.targets(509,5): error MSB3202: The project file "C:\Users\v-reinawang\Aspire项目1\Aspireé¡¹ç›®1.AppHost\Aspireé¡¹ç›®1.AppHost.csproj" was not found.
C:\Program Files\dotnet\sdk\9.0.302\NuGet.targets(509,5): error MSB3202: The project file "C:\Users\v-reinawang\Aspire项目1\Aspireé¡¹ç›®1.ServiceDefaults\Aspireé¡¹ç›®1.ServiceDefaults.csproj" was not found.
C:\Program Files\dotnet\sdk\9.0.302\NuGet.targets(509,5): error MSB3202: The project file "C:\Users\v-reinawang\Aspire项目1\Aspireé¡¹ç›®1.ApiService\Aspireé¡¹ç›®1.ApiService.csproj" was not found.
C:\Program Files\dotnet\sdk\9.0.302\NuGet.targets(509,5): error MSB3202: The project file "C:\Users\v-reinawang\Aspire项目1\Aspireé¡¹ç›®1.Web\Aspireé¡¹ç›®1.Web.csproj" was not found.
Restore failed.

--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -13,8 +13,8 @@
     <DotNetRuntimeCurrentVersionForTesting>9.0.6</DotNetRuntimeCurrentVersionForTesting>
     <!-- dotnet 8.0 versions for running tests - used for templates tests -->
     <DotNetSdkPreviousVersionForTesting>8.0.406</DotNetSdkPreviousVersionForTesting>
-    <!-- dotnet 10.0 versions for running tests - used for templates tests -->
-    <DotNetSdkCurrentVersionForTesting>9.0.200</DotNetSdkCurrentVersionForTesting>
+    <!-- dotnet 9.0 versions for running tests - used for templates tests -->
+    <DotNetSdkCurrentVersionForTesting>9.0.304</DotNetSdkCurrentVersionForTesting>
     <XunitV3Version>3.0.0</XunitV3Version>
     <XUnitAnalyzersVersion>1.21.0</XUnitAnalyzersVersion>
     <XunitRunnerVisualStudioVersion>3.1.0</XunitRunnerVisualStudioVersion>

--- a/src/Aspire.ProjectTemplates/templates/aspire-empty/9.3/AspireApplication.1.sln
+++ b/src/Aspire.ProjectTemplates/templates/aspire-empty/9.3/AspireApplication.1.sln
@@ -1,4 +1,4 @@
-Microsoft Visual Studio Solution File, Format Version 12.00
+﻿Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 17
 VisualStudioVersion = 17.8.0.0
 MinimumVisualStudioVersion = 17.8.0.0

--- a/src/Aspire.ProjectTemplates/templates/aspire-empty/9.4/AspireApplication.1.sln
+++ b/src/Aspire.ProjectTemplates/templates/aspire-empty/9.4/AspireApplication.1.sln
@@ -1,4 +1,4 @@
-Microsoft Visual Studio Solution File, Format Version 12.00
+﻿Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 17
 VisualStudioVersion = 17.8.0.0
 MinimumVisualStudioVersion = 17.8.0.0

--- a/src/Aspire.ProjectTemplates/templates/aspire-starter/9.3/Aspire-StarterApplication.1.sln
+++ b/src/Aspire.ProjectTemplates/templates/aspire-starter/9.3/Aspire-StarterApplication.1.sln
@@ -1,4 +1,4 @@
-Microsoft Visual Studio Solution File, Format Version 12.00
+﻿Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 17
 VisualStudioVersion = 17.8.0.0
 MinimumVisualStudioVersion = 17.8.0.0

--- a/src/Aspire.ProjectTemplates/templates/aspire-starter/9.4/Aspire-StarterApplication.1.sln
+++ b/src/Aspire.ProjectTemplates/templates/aspire-starter/9.4/Aspire-StarterApplication.1.sln
@@ -1,4 +1,4 @@
-Microsoft Visual Studio Solution File, Format Version 12.00
+﻿Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 17
 VisualStudioVersion = 17.8.0.0
 MinimumVisualStudioVersion = 17.8.0.0

--- a/tests/Aspire.Templates.Tests/TemplateTestsBase.cs
+++ b/tests/Aspire.Templates.Tests/TemplateTestsBase.cs
@@ -270,12 +270,7 @@ public partial class TemplateTestsBase
         {
             // Avoid running these cases on PR validation
 
-            if (!OperatingSystem.IsWindows())
-            {
-                // ActiveIssue for windows: https://github.com/dotnet/aspire/issues/4555
-                yield return "aspire_龦唉丂荳_㐁ᠭ_ᠤསྲིདخەلꌠ_1ᥕ";
-            }
-
+            yield return "aspire_龦唉丂荳_㐁ᠭ_ᠤསྲིདخەلꌠ_1ᥕ项目1"; // sln should have UTF-8 byte order mark
             yield return "aspire_starter.1period then.34letters";
             yield return "aspire-starter & with.1";
 

